pom.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.openshift</groupId>
+    <artifactId>booster-parent</artifactId>
+    <version>10</version>
+  </parent>
+
+  <groupId>io.openshift.booster</groupId>
+  <artifactId>wfswarm-cache-booster</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <properties>
+    <version.wildfly.swarm>2018.3.3</version.wildfly.swarm>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <version.arquillian.cube>1.9.1</version.arquillian.cube>
```

Comment:
Please remove this and the `arquillian-cube-bom` from `<dependencyManagement>`, this is already brought in by latest `booster-parent`.

Reply:
done